modules/swagger-codegen/src/test/java/io/swagger/codegen/AbstractIntegrationTest.java

```diff
@@ -21,7 +21,7 @@
     protected abstract Map<String, String> configProperties();
 
     // @wing328: ignore for the time being until we fix the error with the integration test
-    @Test(enabled = false)
+    @Test(enabled = true)
```

Comment:
@wing328 can you check again if these tests are still a problem? They run fine on my machine an i think they have a nice added value.